PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Thank you for pull request! If you are proposing a change to one of the lesson,
```

Comment:
Suggestion:
Thank you **for submitting a** pull request! If you are proposing a change to one of the **lessons**,